src/components/MainScreen.tsx

```diff
@@ -0,0 +1,90 @@
+import { BodyText } from 'components/Text'
+import Button from 'components/Button'
+import classnames, {
+  alignItems,
+  backgroundImage,
+  blur,
+  display,
+  flex,
+  flexDirection,
+  gap,
+  gradientColorStops,
+  inset,
+  justifyContent,
+  margin,
+  maxHeight,
+  minHeight,
+  minWidth,
+  objectFit,
+  opacity,
+  overflow,
+  position,
+  zIndex,
+} from 'classnames/tailwind'
+
+const logo = classnames(margin('my-20'))
+const container = classnames(
+  display('flex'),
+  flex('flex-1'),
+  flexDirection('flex-col'),
+  justifyContent('justify-center'),
+  alignItems('items-center'),
+  zIndex('z-20')
+)
+const text = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  justifyContent('justify-center'),
+  alignItems('items-end'),
+  gap('gap-4')
+)
+const buttonContainer = classnames(margin('my-10'))
+const videoContainer = classnames(
+  position('absolute'),
+  inset('inset-0'),
+  blur('blur-xl'),
+  minWidth('min-w-full'),
+  minHeight('min-h-full'),
+  maxHeight('max-h-full'),
+  overflow('overflow-hidden'),
+  objectFit('object-cover'),
+  zIndex('-z-1')
+)
+const blueOverlay = classnames(
+  position('absolute'),
+  inset('inset-0'),
+  backgroundImage('bg-gradient-to-b'),
+  gradientColorStops('from-blue-start', 'to-blue-end'),
+  zIndex('z-10'),
+  opacity('opacity-70')
+)
+
+export default function () {
+  return (
+    <>
+      <video
```

Comment:
I've put video here, because it takes too much time to load it on `/email` page which is not needed